src/hal/drivers/hal_gm.c

```diff
@@ -1126,18 +1126,18 @@ read(void *arg, long period)
 {
     	gm_device_t	*device = (gm_device_t *)arg;
     	card	*pCard = device->pCard;
-    	int		i;
 	hal_u32_t temp;
-	
+	unsigned int i;
```

Comment:
Why do you change location of the declaration?

Reply:
I wish you had asked me a few months ago. Looking at it now I feel like i may have ordered the variables by their "localness", i.e. these are declared function-wide but used over different regions of the code and i I may have perceived as the most local.